rust/src/builder.rs

```diff
@@ -153,6 +171,17 @@ impl DeltaTableBuilder {
         self
     }
 
+    /// Sets `log_buffer_size` to the builder
+    pub fn with_buffer(mut self, log_buffer_size: usize) -> DeltaResult<Self> {
```

Comment:
```suggestion
    pub fn with_log_buffer_size(mut self, log_buffer_size: usize) -> DeltaResult<Self> {
```

Reply:
👍 I think this is better as it's more explicit